lib/internal/test_runner/coverage.js

```diff
@@ -177,6 +186,12 @@ class TestCoverage {
         if (j > 0 && ranges.length > 0) {
           const range = ranges[0];
 
+          ArrayPrototypePush(functionReports, {
+            name: functions[j].functionName,
```

Comment:
```suggestion
            __proto__: null,
            name: functions[j].functionName,
```

Reply:
FYI, there is eslint rule that will automatically add it to the test runner code